tests/quantization/test_compilation.py

```diff
@@ -26,6 +26,7 @@
 INPUT_OUTPUT_FEATURE = [1, 2, 3]
 
 
+@pytest.mark.flaky
```

Comment:
please create an issue and add a FXIME here if this PR has detected a new flaky issue

Reply:
are you sure this is the right issue ? https://github.com/zama-ai/concrete-ml-internal/issues/4169 talks about simulation but this test is mostly about compilation (which made a worker crash in the last CI)

Reply:
Yes this is the same issue (see https://github.com/zama-ai/concrete-ml/actions/runs/7129808481/job/19414823145#step:27:30139 linked in the issue)